cbindgen 0.29.1 failing to compile?

anyone else running into this after 0.29.1 went out? we just depend on `cbindgen = "0.29"` and `cargo c` now failing with:
```
error[E0433]: failed to resolve: could not find `Content` in `de`
  --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cbindgen-0.29.1/src/bindgen/cargo/cargo_metadata.rs:41:17
   |
41 | #[derive(Clone, Deserialize, Debug)]
   |                 ^^^^^^^^^^^ could not find `Content` in `de`
   |
note: found an item that was configured out
  --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/serde-1.0.228/src/private/de.rs:14:21
   |
12 | #[cfg(any(feature = "std", feature = "alloc"))]
   |          ------------------------------------ the item is gated here
13 | pub use self::content::{
14 |     content_as_str, Content, ContentDeserializer, ContentRefDeserializer, ContentVisitor,
   |                     ^^^^^^^
   = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: could not find `ContentVisitor` in `de`
  --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cbindgen-0.29.1/src/bindgen/cargo/cargo_metadata.rs:41:17
   |
41 | #[derive(Clone, Deserialize, Debug)]
   |                 ^^^^^^^^^^^ could not find `ContentVisitor` in `de`
   |
note: found an item that was configured out
  --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/serde-1.0.228/src/private/de.rs:14:75
   |
12 | #[cfg(any(feature = "std", feature = "alloc"))]
   |          ------------------------------------ the item is gated here
13 | pub use self::content::{
14 |     content_as_str, Content, ContentDeserializer, ContentRefDeserializer, ContentVisitor,
   |                                                                           ^^^^^^^^^^^^^^
   = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)

    Checking toml_writer v1.0.4
error[E0412]: cannot find type `Content` in module `_serde::__private228::de`
  --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cbindgen-0.29.1/src/bindgen/cargo/cargo_metadata.rs:41:17
   |
41 | #[derive(Clone, Deserialize, Debug)]
   |                 ^^^^^^^^^^^ not found in `_serde::__private228::de`
   |
note: found an item that was configured out
  --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/serde-1.0.228/src/private/de.rs:14:21
   |
12 | #[cfg(any(feature = "std", feature = "alloc"))]
   |          ------------------------------------ the item is gated here
13 | pub use self::content::{
14 |     content_as_str, Content, ContentDeserializer, ContentRefDeserializer, ContentVisitor,
   |                     ^^^^^^^
   = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: could not find `ToString` in `__private228`
  --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cbindgen-0.29.1/src/bindgen/cargo/cargo_metadata.rs:41:17
   |
41 | #[derive(Clone, Deserialize, Debug)]
   |                 ^^^^^^^^^^^ could not find `ToString` in `__private228`
   |
   = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider importing this trait
   |
13 + use std::string::ToString;
   |

error[E0433]: failed to resolve: could not find `Vec` in `__private228`
  --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cbindgen-0.29.1/src/bindgen/cargo/cargo_metadata.rs:41:17
   |
41 | #[derive(Clone, Deserialize, Debug)]
   |                 ^^^^^^^^^^^ could not find `Vec` in `__private228`
   |
   = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider importing this struct
   |
13 + use std::vec::Vec;
   |

error[E0425]: cannot find function, tuple struct or tuple variant `FlatMapDeserializer` in module `_serde::__private228::de`
    --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cbindgen-0.29.1/src/bindgen/cargo/cargo_metadata.rs:41:17
     |
  41 | #[derive(Clone, Deserialize, Debug)]
     |                 ^^^^^^^^^^^ not found in `_serde::__private228::de`
     |
note: found an item that was configured out
    --> /Users/zach.schuermann/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/serde-1.0.228/src/private/de.rs:3184:12
     |
3183 | #[cfg(any(feature = "std", feature = "alloc"))]
     |          ------------------------------------ the item is gated here
3184 | pub struct FlatMapDeserializer<'a, 'de: 'a, E>(
     |            ^^^^^^^^^^^^^^^^^^^
     = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/CHANGES
+++ b/CHANGES
@@ -1,5 +1,9 @@
 # unreleased
 
+# 0.29.2
+
+      * Request serde's std feature to avoid issues with newer toml versions.
+
 # 0.29.1
 
       * cf13c2b enum: Track dependencies properly in enumerations.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "cbindgen"
-version = "0.29.1"
+version = "0.29.2"
 authors = [
   "Emilio Cobos Álvarez <emilio@crisal.io>",
   "Jeff Muizelaar <jmuizelaar@mozilla.com>",
@@ -22,7 +22,7 @@ exclude = [
 clap = { version = "4.3", optional = true }
 indexmap = "2.1.0"
 log = "0.4"
-serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+serde = { version = "1.0.103", default-features = false, features = ["derive", "std"] }
 serde_json = "1.0"
 tempfile = "3"
 toml = { version = "0.9", default-features = false, features = ["parse", "serde", "std"] }

